### PR TITLE
[New Term] C++ Math Functions `remquo()`

### LINKED ISSUE #988: [Term Entry] C++ Math Functions: `remquo()`
### Reason/inspiration (optional)

Under `cpp/concepts/math-functions/terms/`, create a new term entry for the `remquo()` function.

- In the terms/ folder, make a new folder called `remquo`
- Inside the `remquo` folder, create a new markdown file called `remquo.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Let's include an "Example" section with a \```cpp code snippet.
- Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ddiegow**
### Description

Create remquo entry.  Closes #988 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/math-functions/terms/remquo/remquo.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+## Syntax
+
+```pseudo
+remquo(numerator, denominator, int* quotient)
+```
+
+The [data type](https://www.codecademy.com/resources/docs/cpp/data-types) of the return value will either be a `double`, `float`, or `long double`. Combinations of these types will return a `double`. The parameter `quotient` must always be an `int` pointer.
+
+The remquo function calculates the floating-point remainder `f` of `numerator / denominator` such that `numerator = n * denominator + f*`, where n is an integer, f has the same sign as x, and the absolute value of f is less than the absolute value of y.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/remquo/remquo.md:25]:
```suggestion
The remquo function calculates the floating-point remainder `f` of `numerator / denominator` such that `numerator = n * denominator + f*`, where n is an integer, `f` has the same sign as `numerator`, and the absolute value of `f` is less than the absolute value of `denominator`.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ddiegow I did a first review and only had one comment.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@ddiegow I've added a second review.

**Review by KTom101 — APPROVED** [review]
@ddiegow This looks good, thank you for making the changes. 

@SSwiniarski Did you have any additional changes you would like to make to this?

**Review by SSwiniarski — APPROVED** [review]
@ddiegow Looks good to me.

**Comment by KTom101** [comment]
@ddiegow Thank you for contributing. The entry is live here: https://www.codecademy.com/resources/docs/cpp/math-functions/remquo

I noticed you are listed as an anonymous contributor:
Screen Shot 2023-01-08 at 7 50 02 PM

If you want to be listed as a contributor, please follow the steps below to add your GitHub username to your Codecademy Profile:

1.  Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile"  in the drop-down box:

Screen Shot 2023-01-08 at 7 56 27 PM

2. Then log out and log back into Codecademy using your GitHub login:

Log in to Codecademy

Let me know if you need any help if you decide to do so.

**Pull request merged**